README.md

```diff
@@ -21,15 +21,15 @@ Make sure you have Docker [installed](https://docs.docker.com/engine/install/) a
 git clone https://github.com/aeternity/hyperchains_playground.git
 ```
 
-### 2. Setup activation criteria:
+### 2. Setup activation criteria (optional):
```

Comment:
It's not

Reply:
I made it optional by adding `stake: 1000` to config files as the default value. @erlmachinedev